elements/lisk-p2p/test/integration/cleanup_unresponsive_peers.ts

```diff
@@ -28,6 +28,10 @@ describe('Cleanup unresponsive peers', () => {
 		...new Array(NETWORK_PEER_COUNT).keys(),
 	].map(index => NETWORK_START_PORT + index);
 
+	before(async () => {
+		sandbox.restore();
```

Comment:
`sandbox.restore` should be called from the global hook?

Reply:
@shuse2 there are some cases that we have to clean sandbox manually otherwise the execution hang. Here you go and example https://github.com/LiskHQ/lisk-sdk/blob/f54569612472ce3dd4336ffe918f9160b93c2a5c/elements/lisk-p2p/test/unit/peer/base.ts#L80 . Does anyone think of any better workaround?

Reply:
@diego-G restoring in before instead of after works